tools/importer/rules/zPattern.js

```diff
@@ -0,0 +1,62 @@
+
+
+
+export default function guessColumnsBlocks(main, document) {
+    const z_pattern_id = ['root_content_positionunwrapped_11',
+      'riverflow',
+      'root_content_positionunwrapped_118916167',
+      'root_content_positionunwrapped_11_245336210',
+      'root_content_position_position-par_positionunwrapped_329391457',
+      'root_content_positionunwrapped_343631879',
+      'id-4410a78dae2d03d01201fa5e2cadc367',
+      'id-4ea65f9011f55f2f6372d3d170674159',
+      'root_content_positionunwrapped_880712250',
+      'root_content_positionunwrapped_11_1515817641',
+      'root_content_flex_1267731132',
+      'id-bebcc5a926005d87942d62277d1ea011',
+      'id-a5d0a62c3b5ba595e2bb739f43c3e216',
+    ];
+  
+    for (var i = 0; i < z_pattern_id.length; i++) {
+      let containers = [];
+      let tempcontainers = [...document.body.querySelectorAll(`#${z_pattern_id[i]} .dexter-FlexContainer-Items`)].filter((c) => {
```

Comment:
`let tempcontainers = [...main.querySelectorAll(`#${z_pattern_id[i]} .dexter-FlexContainer-Items`)].filter((c) => {`
instead of document use main here.
`main` to select element
`document` to create elements

Reply:
done